TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift

```diff
@@ -102,17 +117,15 @@ struct KeychainQuery: KeychainQueryProtocol {
     properties([
       kSecClass: kSecClassGenericPassword,
       kSecAttrAccount: key,
-      kSecReturnData: true,
-      kSecAttrAccessible: kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly
```

Comment:
Just to confirm, we don't the `attrAccessible` for getting a value? Will iOS control the access by itself?

Reply:
Yes, it is only important when saving an element, the other attributes works as filters during searching or deleting elements. Removing this filter will allow to get or delete any element with any of the two supported `attrAccessible`.